tests/page-objects/default/enketo/common-enketo.wdio.page.js

```diff
@@ -27,8 +27,9 @@ const getCorrectFieldsetSection = async (section) => {
   return section;
 };
 
-const isElementDisplayed = async (type, text) => {
-  return await (await getCurrentPageSection()).$(`${type}*=${text}`).isDisplayed();
+const isElementDisplayed = async (type, text, parent) => {
+  const elemSelector = parent === undefined ? getCurrentPageSection() : $(`${parent}`);
+  return await (await elemSelector).$(`${type}*=${text}`).isDisplayed();
```

Comment:
```suggestion
const isElementDisplayed = async (type, text) => {
  return await (await getCurrentPageSection()).$(`${type}*=${text}`).isDisplayed();
```

I am sorry if [this comment](https://github.com/medic/cht-core/pull/9498#discussion_r1803520476) was confusing, I didn't mean to modify the method in the `common-enketo.wdio.page.js`, that method is to verify if the specific element is displayed or not but inside the enketo forms, we should keep the method as it was, I was just trying to give an example of how you can create the method for tasks, sorry for the confusion.

Please create a method in `tasks.wdio-spec.js` that verifies if the loading status message is displayed, something like this:

```.js
const isNoMoreTasksMessageDisplayed = async () => {
  return await (await $(TASK_LIST_SELECTOR)).$('p*=No more tasks').isDisplayed();
}
```

_PS: You can change the name to the method if you want, I didn't come up with a better one_ 😅

Reply:
Gotcha